src/test/coinselection_tests.cpp

```diff
@@ -0,0 +1,41 @@
+// Copyright (c) 2011-2013 The Bitcoin Core developers
+// Distributed under the MIT software license, see the accompanying
+// file COPYING or http://www.opensource.org/licenses/mit-license.php.
+
+//
+// Unit tests for coinselection algorithm in wallet
+//
+
+#include "uint256.h"
+#include "test/test_bitcoin.h"
+#include "wallet.cpp"
+
+#include <boost/test/unit_test.hpp>
+
+using namespace std;
+
+BOOST_FIXTURE_TEST_SUITE(coinselection_tests, BasicTestingSetup)
+
+BOOST_AUTO_TEST_CASE(sanity)
```

Comment:
Ah thanks, I had been wondering why there was no wallet tests in the regular test directory.